ci-operator/config/openstack-k8s-operators/keystone-operator/openstack-k8s-operators-keystone-operator-master.yaml

```diff
@@ -16,6 +16,10 @@ build_root:
 images:
 - dockerfile_literal: |
     FROM src
+    RUN curl -s -L "https://github.com/kudobuilder/kuttl/releases/download/v0.15.0/kubectl-kuttl_0.15.0_linux_x86_64" -o kubectl-kuttl
```

Comment:
I have proposed a patch[5] to add kuttl binaries in our build_root image.  Once https://github.com/openstack-k8s-operators/openstack-k8s-operators-ci/pull/72 merges we don't need to add these here.

Reply:
My proposal is to continue using this Dockerfile to create the image and not updating root image, as mentioned in the oko-ci patch.
Lets keep this here, we can add:
`ARG KUTTL_VERSION=v0.15.0
`and 
`RUN curl -s -L "https://github.com/kudobuilder/kuttl/releases/download/${KUTTL_VERSION}/kubectl-kuttl_${KUTTL_VERSION#v}_linux_x86_64" -o kubectl-kuttl`
to make easier to update its version across all files in this repo.
With that, we can at least test newer versions of kuttl bins easily.